design/src/main/kotlin/uk/govuk/app/design/ui/component/List.kt

```diff
@@ -96,6 +98,43 @@ fun ExternalLinkListItem(
                 tint = GovUkTheme.colourScheme.textAndIcons.link
             )
         }
+
+        ShowSubText(subText)
+    }
+}
+
+@Composable
+private fun applyPadding(subText: String): PaddingValues {
```

Comment:
We could simplify this and avoid creating and reassigning local variable? 

`@Composable
private fun applyPadding(subText: String): PaddingValues {
    if (subText.isNotEmpty()) {
        return PaddingValues(
            top = GovUkTheme.spacing.medium,
            start = GovUkTheme.spacing.medium,
            end = GovUkTheme.spacing.medium,
            bottom = 0.dp
        )
    }
    return PaddingValues(all = GovUkTheme.spacing.medium)
}`

Reply:
You can actually pull the return up into the if:

`return if (subText.isNotEmpty()) { 
        PaddingValues(
            top = GovUkTheme.spacing.medium,
            start = GovUkTheme.spacing.medium,
            end = GovUkTheme.spacing.medium,
            bottom = 0.dp
        )
    } else {
        PaddingValues(all = GovUkTheme.spacing.medium)
    }`

Reply:
Thanks both - great suggestions 👏 

Sorry @TimGJD but I like @davidc-gds `return if` a bit better, so went for that.